spec/html-spec.coffee

```diff
@@ -611,3 +611,22 @@ describe 'HTML grammar', ->
     it "tolerates hyphens in other tag names", ->
       lines = grammar.tokenizeLines '<foo-bar>'
       expect(lines[0][1]).toEqual value: 'foo-bar', scopes: ['text.html.basic', 'meta.tag.other.html', 'entity.name.tag.other.html']
+
+  describe "snippets", ->
+    snippetsModule = null
+
+    beforeEach ->
+      waitsForPromise ->
+        atom.packages.activatePackage('snippets').then (p) -> snippetsModule = p.mainModule
+
+      runs ->
+        # Do not load user snippets
+        spyOn(snippetsModule, 'loadUserSnippets').andCallFake (callback) -> callback({})
+
+      waitsFor 'snippets to load', (done) -> snippetsModule.onDidLoadSnippets(done)
+
+    it "suggests snippets", ->
+      expect(Object.keys(snippetsModule.parsedSnippetsForScopes(['.text.html'])).length).toBeGreaterThan 10
+
+    xit "does not suggest any HTML snippets when in embedded scripts", ->
```

Comment:
This test _should_ return 0, however it's returning 8.  The following snippets are still getting through, and I have no idea why:
* favicon
* hgroup
* import
* math
* option
* rb
* rtc
* svg

Reply:
Could it be due to the fact that those snippets are present in [this section](https://github.com/atom/language-html/blob/0c7d873e659ce6b8a46c86844ef09c3b969d96fe/snippets/language-html.cson#L1-L389) of `language-html.cson` but not present in [this section](https://github.com/atom/language-html/blob/0c7d873e659ce6b8a46c86844ef09c3b969d96fe/snippets/language-html.cson#L390-L646)?

Reply:
Ah!  Fantastic catch, I didn't notice that there was a separate snippets section.  Great :eyes:!